<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1455</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>Cascadia Mono SemiLight</family>
       </font>
      </property>
      <property name="title">
       <string>Create</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ez::protocol_serializer ps({</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CreatorWidget" name="creatorWidget" native="true"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>});</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>884</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <family>Cascadia Mono SemiLight</family>
       </font>
      </property>
      <property name="title">
       <string>Edit</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="EditorWidget" name="editorWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="font">
       <font>
        <family>Cascadia Mono SemiLight</family>
       </font>
      </property>
      <property name="title">
       <string>Visualize</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="VisualizerWidget" name="visualizerWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CreatorWidget</class>
   <extends>QWidget</extends>
   <header>creator_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditorWidget</class>
   <extends>QWidget</extends>
   <header>editor_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VisualizerWidget</class>
   <extends>QWidget</extends>
   <header>visualizer_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
